soaks/tests/socket_to_socket_blackhole/README.md

```diff
@@ -0,0 +1,10 @@
+# Socket -> Socket
+
+This soak tests the socket source feeding directly into the socket sink. No
+transformation is done in the middle so throughput is limited solely by
+socket source's ability to create it, and socket sink's ability to absorb it.
+
+## Method
+
+Lading `tcp_gen` is used to generate socket load into vector, `socket` acts as a
+sink.
```

Comment:
Nit: mention `tcp_blackhole` as the destination?

Reply:
Added in 40378a336d80f1e278247bba938d09e6d7a84575